### PR TITLE
Issue 5

### PR DESCRIPTION
**Pull request by RainierBarrett**
This addresses the format specifications of HOOMD-blue.

### CODE REVIEW

File: tensorflow_plugin/models/example-models/ann-learned/run_train.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
     #equilibrate for 4k steps first
     hoomd.run(4000)
     #now attach the trainable model
-    tfcompute.attach(nlist, r_cut=rcut, save_period=10, period=100, feed_func=lambda x: {'keep_prob:0': 0.8})
+    tfcompute.attach(nlist, r_cut=rcut, save_period=10, period=100, feed_dict={'keep_prob:0': 0.8})
```
Comment (whitead) [inline tensorflow_plugin/models/example-models/ann-learned/run_train.py]: Why is this part of the commit?

Comment (RainierBarrett) [reply tensorflow_plugin/models/example-models/ann-learned/run_train.py]: Oops. I had to change this after merging from master to get it to run with some changes from there. This is not related to this issue.

File: tensorflow_plugin/CommStruct.h [hunk 2]
```diff
@@ -12,139 +12,168 @@
...
+            ArrayHandle<T> handle(*_array, access_location::device, access_mode::read);
+            cudaMemcpy(dest, handle.data, n, cudaMemcpyDeviceToDevice);
+            }
+        void write_gpu_memory(const void* src, size_t n) override
+            {
+            ArrayHandle<T> handle(*_array, access_location::device, access_mode::overwrite);
+            cudaMemcpy(handle.data, src, n, cudaMemcpyDeviceToDevice);
+            }
+#else
+        void read_gpu_memory(void *dest, size_t n) override
```
[the remark this answers is not in the document]
Comment (RainierBarrett) [reply tensorflow_plugin/CommStruct.h]: The original declarations have a comment that just states exactly what they do. I didn't comment on the overrides specifically. Should we?

**Comment by whitead** [comment]
If you can run doxygen and double check that everything works, we can merge this in.

**Comment by RainierBarrett** [comment]
I ran doxygen and it works, but there are some .py files still lacking documentation. I'll work on those after my other 0.2 issues if I get time. Should we commit the generated html files?

**Comment by whitead** [comment]
Let's not commit the html for now. Open a new issue for the python doc so we don't forget.

**Pull request merged**